Chapter02/recipe-0011/README.md

```diff
@@ -0,0 +1,100 @@
+# Discovering the operating system
+
+CMake is a set of cross-platform tools. Nevertheless it can be very useful to
+know on which operating system (OS) the configure or build step is executed:
+either to tweak CMake code for a particular OS, to enable conditional
+compilation depending on the OS or to use compiler-specific extensions if
+available. In this recipe we will demonstrate how to use CMake to detect the
+OS.
+
+
+## Ingredients
```

Comment:
I had started using ingredients, but the start-up guide to writing the cookbook is quite clear about the naming of the subsections:
- Getting ready, optional
- How to do it, mandatory
- How it works, mandatory
- There's more, optional

We need to ask editors if there's any leeway for the optional subsections.

Reply:
Fair enough. Will adapt.